app/src/main/res/values/strings.xml

```diff
@@ -59,7 +59,7 @@
   <string name="add_set_name_toast">Please provide a name for this set</string>
   <string name="provider_modifications">Modifications</string>
   <string name="menu_upload_single">Upload</string>
-  <string name="categories_search_text_hint">Search categories</string>
```

Comment:
@vanshikaarora looks like removing this line is causing the build to fail, the string **categories_search_text_hint** is already in use at [activity_upload_categories.xml](https://github.com/vanshikaarora/apps-android-commons/blob/upload_caption_depict/app/src/main/res/layout/activity_upload_categories.xml)

Reply:
Thanks for catching that!
Indeed both are needed, the app will have **both** categories and "depicts".

Reply:
Thanks for checking that @madhurgupta10 :)